ui/app/components/ui/boxed-message/boxed-message.component.js

```diff
@@ -0,0 +1,55 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+
+export default class BoxedMessage extends PureComponent {
```

Comment:
Is having a separate `BoxedMessage` component useful? Particularly one without any stories. I can't think of any cases offhand where this might be reused - perhaps it would be simpler to consolidate these two components into one.

Reply:
Well, really this is a more basic version of the notifications we show on the home screen when, for example, reminding the user to back up their seed phrase or notifying about detected 3box backup. Or perhaps I mean to say that there is an abstraction common to `ui/app/components/app/home-notification` and `ui/app/components/ui/alert-message`

Now that we have notifications of similar design in two different places, my sense is that there will be others.

But I imagine you'd prefer we simplify this until that abstraction is actually used?

Reply:
I'm not sure those cases are the same. Would they also have an Icon associated with them for example?

If there is a common abstraction between these two, which there might be (the styles are similar), I don't think this is it. If that was the goal, I'd suggest actually using this component in those places from the start. If that's not done, I don't think this is useful as a distinct component.

Reply:
Sorry I should have clarified as well that this concern is non-blocking.

Reply:
With 2d4b2ed5d84a48f0d1345bb53d29db405f003e5e `BoxedMessage` is used within `HomeNotification` component. Home notification component is unchanged:

![Screenshot from 2020-03-30 10-28-10](https://user-images.githubusercontent.com/7499938/77915144-8535fc00-7271-11ea-8deb-93276120b31d.png)


This shows how `BoxedMessage` covers some of the commonalities between the home notifcations and these toast messages:

![Screenshot from 2020-03-30 10-31-20](https://user-images.githubusercontent.com/7499938/77915195-9bdc5300-7271-11ea-836d-e3ef1b6c75e1.png)

The need for a component that handles common abstractions is further seen when looking at this design from the figma error messaging designs

![Screenshot from 2020-03-30 10-14-59](https://user-images.githubusercontent.com/7499938/77915280-c4fce380-7271-11ea-8614-a8b6d27ee19f.png)

The toast their has the icon and description text of the `ToastMessage`, but clickable buttons like the `HomeNotification`. The `BoxedMessage` component could easily be extended to provide the buttons, so that logic could be abstracted from the `HomeNotification` component, and then used when implementing that error message. Those changes could be made in the future, at the time of implementing the error message.

Reply:
Ah, I hadn't realized that all of the "boxed messages" we use had this two-column layout. I thought some of them lacked icons, or had the icons inline with the text, but I can't find examples of either of those now.

This seems pretty reasonable!